scripts/provision.sh

```diff
@@ -46,6 +46,7 @@ grafana,44564533c9d4d656bdcd5633b808f0bf6fb177ae,https://github.com/vito/grafana
 logsearch,23.0.0,https://bosh.io/d/github.com/logsearch/logsearch-boshrelease?v=23.0.0
 logsearch-for-cloudfoundry,7,https://logsearch-for-cloudfoundry-boshrelease.s3.amazonaws.com/boshrelease-logsearch-for-cloudfoundry-7.tgz
 redis,420,https://bosh.io/d/github.com/pivotal-cf/cf-redis-release?v=420
+nginx,667cc1a0f9117bdb4b217ee2b76dc20e61371c02,https://s3.amazonaws.com/nginx-release/nginx-2.tgz
```

Comment:
Is it worth a note in the commit that this comes from https://github.com/cloudfoundry-community/nginx-release ? I found it by Googling the URL, but it would be good to be explicit.